vignettes/individual-claims.Rmd

```diff
@@ -0,0 +1,57 @@
+---
+title: "Individual Claims"
+output: rmarkdown::html_vignette
+vignette: >
+  %\VignetteIndexEntry{individual-claims}
+  %\VignetteEngine{knitr::rmarkdown}
+  %\VignetteEncoding{UTF-8}
+---
+
+```{r, include = FALSE}
+knitr::opts_chunk$set(
+  collapse = TRUE,
+  comment = "#>"
+)
+```
+
+This article outlines the conventions for simulators that generate individual claims data.
+
+#### Data Format
+
+Each row should represent the state of a claim at one point in time.
```

Comment:
If the goal is to simulate claims over their life cycle, then I think we should actually have event dates instead of accident_year and development_year. Also, is there a preference for incremental or cumulative values? Depending on the simulation methodology one or the other will be more "natural". Does it make sense to have this information encoded as an attribute (i.e., would be a subclass of tibble).

Reply:
Different simulators operate at different level of granularity, so some only go down to accident year/quarter. I guess if there are event dates, AY would be considered redundant. As for incremental vs. cumulative, I feel like incremental is slightly "easier" to work with and most algorithms model the incremental distribution, but wonder if others have strong opinions otherwise.

Reply:
I agree in simulating incremental values. Mainly for these reasons:
1. Incremental values can be considered (under certain assumptions) independent 
2. I think it will be easier to benchmark and check figures against other algorithms currently in use.